### ISSUE TITLE
Dataset Inquiry for Model Training

### ISSUE THREAD

**Issue by ghufran1054** [JWFanggit/LOTVS-CAP#15]
I have got this project in a running condition. I just wanted to ask a couple of things from author or if anybody else has done it:

1) I downloaded the dataset from the hugging Face repo for CAP and DADA-2000. The image size in them is, 1584 x 660 (or something close to these numbers). The model requires 224x224 Input, but I found no Image Resize Transform in your Dataloader. So, I wanted to ask how should I resize these Images to 224x224 so I could reproduce the results in the paper. If I center crop them, I lose a lot of information from width from both sides, If I just resize it to 224x224 the image kind of distorts. So can you please help in this regard.

2) Secondly, How should the training loss begin from the first epoch, I mean what is its approximate starting value, and what's its trend for decreasing ?

**Comment by star-lxh** [comment]
> I have got this project in a running condition. I just wanted to ask a couple of things from author or if anybody else has done it:
> 
> 1. I downloaded the dataset from the hugging Face repo for CAP and DADA-2000. The image size in them is, 1584 x 660 (or something close to these numbers). The model requires 224x224 Input, but I found no Image Resize Transform in your Dataloader. So, I wanted to ask how should I resize these Images to 224x224 so I could reproduce the results in the paper. If I center crop them, I lose a lot of information from width from both sides, If I just resize it to 224x224 the image kind of distorts. So can you please help in this regard.
> 2. Secondly, How should the training loss begin from the first epoch, I mean what is its approximate starting value, and what's its trend for decreasing ?

@ghufran1054 Hi, how did you solve this problem?
